src/Mutate.tsx

```diff
@@ -15,7 +15,7 @@ export interface States<TData, TError> {
   error?: GetState<TData, TError>["error"];
 }
 
-export type MutateMethod<TData> = (data?: string | {}) => Promise<TData>;
+export type MutateMethod<TData, TBody> = (data?: string | TBody) => Promise<TData>;
```

Comment:
`TData` represents a response body so I find `TBody` a little confusing in this case. Can we name it `TRequestBody` instead?